docs/AccessingCarePlanData/AccessingCarePlanData-template.md

```diff
@@ -216,33 +217,34 @@ This sample reads all the activities from the store, and then iterates over the
-			                                           
-			[hkstore saveObject:sample withCompletion:^(BOOL success, NSError * _Nullable error) {
-				// 3. When the collected temperature has been saved into health kit  
-				// Use the saved HKSample object to create a result object and save it to CarePlanStore.
-				// Then each time, CarePlanStore will load the temperature data from HealthKit.
-				OCKCarePlanEventResult *result = [[OCKCarePlanEventResult alloc] initWithQuantitySample:sample quantityStringFormatter:nil unitStringKeys:@{[HKUnit degreeFahrenheitUnit]: @"\u00B0F", [HKUnit degreeCelsiusUnit]: @"\u00B0C",} userInfo:nil];
-
-				[_store updateEvent:assessmentEvent withResult:result state:OCKCarePlanEventStateCompleted completion:^(BOOL success, OCKCarePlanEvent * _Nonnull event, NSError * _Nonnull error) {
-					NSAssert(success, error.localizedDescription);
-					}];
-				}];                                        
-			}];
-		}
-	}             
-	
+```objective-c
+- (void)symptomTrackerViewController:(OCKSymptomTrackerViewController *)viewController didSelectRowWithAssessmentEvent:(OCKCarePlanEvent *)assessmentEvent {
+	NSString *identifier = assessmentEvent.activity.identifier;
+
+    if ([identifier isEqualToString:TemperatureAssessment]) {
+		//1. Present a survey to ask for temperature
+		// ...
+
+		//2. Save the collected temperature into health kit
+		HKHealthStore *hkstore = [HKHealthStore new];
+		HKQuantityType *type = [HKQuantityType quantityTypeForIdentifier:HKQuantityTypeIdentifierBodyTemperature];
+
+		[hkstore requestAuthorizationToShareTypes:[NSSet setWithObject:type] readTypes:[NSSet setWithObject:type] completion:^(BOOL success, NSError * _Nullable error) {
+		HKQuantitySample *sample = [HKQuantitySample quantitySampleWithType:type quantity:[HKQuantity quantityWithUnit:[HKUnit degreeFahrenheitUnit] doubleValue:99.1] startDate:[NSDate date] endDate:[NSDate date]];
+
+		[hkstore saveObject:sample withCompletion:^(BOOL success, NSError * _Nullable error) {
+			// 3. When the collected temperature has been saved into health kit  
+			// Use the saved HKSample object to create a result object and save it to CarePlanStore.
+			// Then each time, CarePlanStore will load the temperature data from HealthKit.
+			OCKCarePlanEventResult *result = [[OCKCarePlanEventResult alloc] initWithQuantitySample:sample quantityStringFormatter:nil unitStringKeys:@{[HKUnit degreeFahrenheitUnit]: @"\u00B0F", [HKUnit degreeCelsiusUnit]: @"\u00B0C",} userInfo:nil];
+
+			[_store updateEvent:assessmentEvent withResult:result state:OCKCarePlanEventStateCompleted completion:^(BOOL success, OCKCarePlanEvent * _Nonnull event, NSError * _Nonnull error) {
+				NSAssert(success, error.localizedDescription);
+				}];
+			}];                                        
+		}];
+	}
+}             
+```
```

Comment:
We potentially want to convert this code snippet to Swift.
I would recommend including it in this PR, or at least tracking it under Issues.
`OCKSample` can serve as a good reference if you choose to do it in this PR.